docs/api/regressor/alphas.rst

```diff
@@ -7,6 +7,14 @@ Regularization is designed to penalize model complexity, therefore the higher th
 
 The AlphaSelection Visualizer demonstrates how different values of alpha influence model selection during the regularization of linear models. Generally speaking, alpha increases the affect of regularization, e.g. if alpha is zero there is no regularization and the higher the alpha, the more the regularization parameter influences the final model.
 
+=================   ==============================
+Visualizer           :class:`~yellowbrick.regressor.alphas.AlphaSelection`
```

Comment:
I think you were right to exclude the `ManualAlphaSelection` visualizer here, both because it's not explicitly documented here, but because it is also a prototype. Having it in the API reference is sufficient.